src/components/auth/auth.controller.js

```diff
@@ -0,0 +1,13 @@
+const AuthService = require('./auth.service');
+
+class AuthController {
+  static register(req) {
```

Comment:
as we discussed, we should use only non-static functions for controllers and servirces